octoprint_auth_ldap/ldap.py

```diff
@@ -28,9 +28,13 @@ def get_client(self, user=None, password=None):
             client = ldap.initialize(uri)
             if self.settings.get([REQUEST_TLS_CERT]):
                 self.logger.debug("Requesting TLS certificate")
+                ldap.set_option(ldap.OPT_X_TLS_REQUIRE_CERT, ldap.OPT_X_TLS_DEMAND)
                 client.set_option(ldap.OPT_X_TLS_REQUIRE_CERT, ldap.OPT_X_TLS_DEMAND)
             else:
+                ldap.set_option(ldap.OPT_X_TLS_REQUIRE_CERT, ldap.OPT_X_TLS_NEVER)
```

Comment:
Why this option duplicate the next line `client.set_option(ldap.OPT_X_TLS_REQUIRE_CERT, ldap.OPT_X_TLS_NEVER)` ?